applications/RANSApplication/custom_elements/convection_diffusion_reaction_element_data/k_epsilon/element_data_utilities.h

```diff
@@ -0,0 +1,51 @@
+//    |  /           |
```

Comment:
I'd rename the file to `k_epsilon_element_data_utilities.h` to make easier going through the code. The  same for the rest (e.g. `k_omega_sst_element_data_utilities.h`).

Reply:
I was thinking having folder structure to not to have proposed `data_containers` folder exploding. But, if inside the subfolders we start using long names again repeating the parent folder, it will end up with really long names for includes and some other stuff. I think I read somehere that windows compilation has some problems with long names. What do you say?